src/components/transactions/TxDetails/TxData/DecodedData/index.tsx

```diff
@@ -59,11 +58,9 @@ export const DecodedData = ({ txData, toInfo }: Props): ReactElement | null => {
     decodedData = <HexEncodedData title="Data (hex-encoded)" hexData={txData.hexData} />
   }
 
-  const isL2Migration = isMigrateToL2TxData(txData, chainInfo?.chainId)
-
   return (
     <Stack spacing={2}>
-      {isDelegateCall && <DelegateCallWarning showWarning={!txData.trustedDelegateCallTarget && !isL2Migration} />}
```

Comment:
We shouldn't hard-code delegate warning exceptions. These contracts must be added as trusted in tx-service (maybe they have been already).

Reply:
This was added because users would see an unexpected delegate call warning when they tried to create their first tx and the migration was included. https://github.com/safe-global/safe-wallet-web/pull/4339

If this hasn't changed I would not remove it.

Reply:
Like I wrote above, we should request to add these contracts as "known" and "trusted", it shouldn't be hard-coded.

Reply:
@usame-algan looks like it [was already added](https://github.com/safe-global/safe-transaction-service/blob/ec11de08be6acf5b110de4a8680f526ae56650ed/safe_transaction_service/contracts/management/commands/setup_safe_contracts.py#L13) so there should be no warning.